### DISCUSSION TITLE
even the python is installed i cant select a kernel after the update of jupyter notebook

### DISCUSSION THREAD

**Discussion by sribhashyamsashanksai** [microsoft/vscode-jupyter#12359] [Questions and Answers]
![image](https://user-images.githubusercontent.com/114094697/206498611-b87d761f-ee08-4b80-a1ed-0e9695efe9d8.png)

i have installed python in my device also pls tell me how to fix these ??

**Comment by DonJayamanne** [comment]
Thanks for filing this issue, please could you
* Confirm you have vscode 1.74 installed and the latest versions of Jupyter and Python extensions installed
* Open a python file, and then use the command `PYhton: Select Interpreter`, verify whether you get installed pythons listed in that quick pick list. 

If that still does not work
Please could you enable logging as follows:
* Open settings & go into `Jupyter->Logging`
* Change the value to `verbose`
* Reload VS Code,
* Attempt to repro the issue & then copy the output from the `Jupyter` output panel.
